quickmamba/gui/colorPicker.py

```diff
@@ -0,0 +1,78 @@
+from PyQt5 import QtCore
+from PyQt5 import QtWidgets
+from PyQt5 import QtQuick
+from PyQt5.QtGui import QCursor, QColor, QGuiApplication
+
+class ColorPickingEventFilter(QtCore.QObject):
+    def __init__(self, screenpicker):
+        QtCore.QObject.__init__(self, screenpicker)
+        self._screenpicker = screenpicker
+
+    def eventFilter(self, QObject, QEvent):
```

Comment:
eventFilter takes 3 arguments, not types...
Should not be called: "QObject, QEvent" but something like "object, event".